src/tailwind.preset.js

```diff
@@ -53,12 +55,23 @@ export default {
           light: '#737373',
         },
       },
+      ringColor: {
+        DEFAULT: '#59a7e8',
+      },
+      ringOpacity: {
+        DEFAULT: '1.0',
+      },
```

Comment:
Even though we use a `box-shadow` to style our focus rings, tailwind still involves the `ringOpacity` value when calculating `--tw-ring-color` and the default value is, annoyingly, `0.5`.

Reply:
It would be useful to capture this in a code comment.